server/schedule/operator/step.go

```diff
@@ -763,3 +845,48 @@ func fastStepWaitDuration(regionSize int64) time.Duration {
 	}
 	return wait
 }
+
+func addNode(id, storeID uint64, useConfChangeV2 bool) *pdpb.RegionHeartbeatResponse {
+	change := &pdpb.ChangePeer{
+		ChangeType: eraftpb.ConfChangeType_AddNode,
+		Peer: &metapb.Peer{
+			Id:      id,
+			StoreId: storeID,
+			Role:    metapb.PeerRole_Voter,
+		},
+	}
+	return createResponse(change, useConfChangeV2)
+}
+
+func addLearnerNode(id, storeID uint64, useConfChangeV2 bool) *pdpb.RegionHeartbeatResponse {
```

Comment:
How about removing these two functions and directly putting the inside logic into GetCmd?

Reply:
it's reused by too steps